src/Reflection/ReflectionFunctionAbstract.php

```diff
@@ -270,17 +273,31 @@ public function isClosure() : bool
         return $this->node instanceof Node\Expr\Closure;
     }
 
-    /**
-     * Is this function deprecated?
-     *
-     * Note - we cannot reflect on internal functions (as there is no PHP source
-     * code we can access. This means, at present, we can only EVER return false
-     * from this function.
-     *
-     * @see https://github.com/Roave/BetterReflection/issues/38
-     */
     public function isDeprecated() : bool
     {
+        $docComment = $this->getDocComment();
+        if ($docComment !== '') {
+            $docBlockFactory = DocBlockFactory::createInstance();
```

Comment:
This dependency isn't available and I don't want to add it. Please do regex match instead.